### DISCUSSION TITLE
Grab message from Stream by sequence number?

### DISCUSSION THREAD

**Discussion by DustinReagan** [nats-io/nats.net.v1#692] [General]
The NATS docs mention fetching a particular message from a stream by the sequence number:

https://docs.nats.io/reference/reference-protocols/nats_api_reference#fetching-from-a-stream-by-sequence

If possible, how can I do this with the nats.net API?

Thanks!

**Comment by scottf** [comment]
Use the JetStream Management function GetMessage https://github.com/nats-io/nats.net/blob/master/src/NATS.Client/JetStream/IJetStreamManagement.cs#L150

https://nats-io.github.io/nats.net/interface_n_a_t_s_1_1_client_1_1_jet_stream_1_1_i_jet_stream_management.html
